tests/Unit/Managers/ProcessManagerTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace Churn\Tests\Unit\Managers;
+
+use Churn\Collections\FileCollection;
+use Churn\Managers\ProcessManager;
+use Churn\Factories\ProcessFactory;
+use Churn\Tests\BaseTestCase;
+use Churn\Configuration\Config;
+
+class ProcessManagerTest extends BaseTestCase
+{
+    /** @test */
+    public function it_can_be_instantiated() 
+    {
+        $this->assertInstanceOf(ProcessManager::class, new ProcessManager);
+    }
+
+    /** @test */
+    public function it_returns_collection_with_same_count_as_number_of_parallel_jobs() 
+    {
+        $numParallelJobs = 3;
+        $processManager = new ProcessManager();
+        $config = Config::createFromDefaultValues();
+        $processFactory = new ProcessFactory($config->getCommitsSince());
+        $collection = $processManager->process(new FileCollection, $processFactory, $numParallelJobs);
+        $this->assertEquals($collection->count(), $numParallelJobs);        
```

Comment:
The test fails. I let you investigate why.

Reply:
Well, the test is wrong: `$collection->count()` should be equal the number of files in `FileCollection`.
In your test `FileCollection` is empty so `$collection->count()` should return `0`.